selection-to-clipboard/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+    "manifest_version": 2,
+    "name": "selection-to-clipboard",
+    "description": "Example of WebExtensionAPI for writing to the [clipboard](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Interact_with_the_clipboard) ",
+    "version": "1.0",
+    "homepage_url": "https://github.com/mdn/webextensions-examples/tree/master/selection-to-clipboard",
+
+    "icons": {
+        "48": "icons/clipboard-48.png"
+    },
+
+    "applications": {
+        "gecko": {
+            "id": "selection-to-clipboard@mozilla.org",
```

Comment:
I think we've been trying to not specify an id in the add-ons, they can be run locally in Firefox and if someone wants to sign them, they can do so without conflicting with anyone else.